tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -716,6 +717,12 @@ def test_options(self):
                 event=audit_log.get_event_id("PROJECT_EDIT"),
             ).exists()
         assert project.get_option("filters:react-hydration-errors", "1")
+        with assume_test_silo_mode(SiloMode.CONTROL):
+            assert AuditLogEntry.objects.filter(
+                organization_id=project.organization_id,
+                event=audit_log.get_event_id("PROJECT_EDIT"),
+            ).exists()
```

Comment:
Q: what does this piece do? It seems duplicated for each option.

Reply:
I admit that I have replicated the test without thinking much about it but I have now removed it since semantically it does not make sense, since no mutation of the project options is happening between the checks.